tests/testthat/test-make-style.r

```diff
@@ -30,13 +30,13 @@ test_that("hexa color regex works", {
                 "foo#123456bar")
 
   for (color in positive) {
-    expect_true(grepl(hash_color_regex, color))
-    expect_true(grepl(hash_color_regex, toupper(color)))
+    expect_true(grepl(crayon:::hash_color_regex, color))
+    expect_true(grepl(crayon:::hash_color_regex, toupper(color)))
```

Comment:
Yes, indeed.  I am in the habit of using `test_dir` and wasn't thinking it wouldn't affect your tests as run with `test_check`.